espnet/nets/pytorch_backend/e2e_asr.py

```diff
@@ -223,13 +223,15 @@ def forward(self, xs_pad, ilens, ys_pad):
 
                 hyp_words = seq_hat_text.split()
                 ref_words = seq_true_text.split()
-                wers.append(editdistance.eval(hyp_words, ref_words) / len(ref_words))
+                word_eds.append(editdistance.eval(hyp_words, ref_words))
+                word_ref_lens.append(len(ref_words))
                 hyp_chars = seq_hat_text.replace(' ', '')
                 ref_chars = seq_true_text.replace(' ', '')
-                cers.append(editdistance.eval(hyp_chars, ref_chars) / len(ref_chars))
+                char_eds.append(editdistance.eval(hyp_chars, ref_chars))
+                char_ref_lens.append(len(ref_chars))
 
-            wer = 0.0 if not self.report_wer else sum(wers) / len(wers)
-            cer = 0.0 if not self.report_cer else sum(cers) / len(cers)
+            wer = 0.0 if not self.report_wer else float(sum(word_eds)) / len(word_ref_lens)
```

Comment:
Is `len(word_ref_lens)` correct? Maybe `sum(word_ref_lens)`?

Reply:
Oh, yes. It's sum(word_ref_lens).